Testbed/Example_ReferenceTestBed_UserGuide.md

```diff
@@ -0,0 +1,730 @@
+# Example Reference Testbed User Guide
```

Comment:
We should reference [this](../Testbed/Example_ReferenceTestBed_UserGuide.md) from the README.md to guide the user through the Repository.

Reply:
Created  a new issue for this #18